lib/addAttrs.js

```diff
@@ -0,0 +1,21 @@
+import find from './find';
+
+function addAttrs(selector, attrs) {
+  if (Array.isArray(selector)) {
```

Comment:
I added a test to send an array to the method.
https://github.com/firstandthird/domassist/pull/20/files#diff-6857e5b1a304c14415231f3cf322e5dcR60